docs/_includes/intrinsic_attributes.adoc

```diff
@@ -0,0 +1,467 @@
+|(not set - graphics are linked, not embedded)
+|
+|<<managing-images>>
+
+|deckjs_theme
+|
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|deckjs_transition
+|
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|docinfo
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|docinfo1
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|docinfo2
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|doctype
+|Set the output document type.
+|article
+|article, book, inline, manpage
+|<<document-types>>
+
+|embedded
+|Is this real? related to inline doctype?
+|(not set)
+|?
+|?
+
+|encoding
+|The encoding of the input and output files, all backends.
+|UTF-8
+|Any valid encoding string
+|-
+
+|example-caption
+|Sets the text used to label EXAMPLES.
+|Example
+|Any (value not used)
+|-
+
+|experimental
+|Enable experimental extensions. 
+Emphasises that they are subject to change and may even be removed in the next version.
+Currently only applies to the interface macros.
+|(not set)
+|Any (value not used)
+|<<user-interface-macros>>
+
+|figure-caption
+|Sets the text used to label figures.
+|Figure
+|Any
+|<<images>>
+
+|hardbreaks
+|Preserve hard line breaks in the input.
+|(not set)
+|Any (value not used)
+|<<line-breaks>>
+
+|icons
+|Chooses icons instead of text for admonitions.
+|(not set)
+|font
+|<<admonition-icons>>
+
+|iconsdir
+|Where admonition icons are stored.
+|./images/icons
+|Path
+|<<icons>>
+
+|idprefix
+|Prefix for auto-generated section IDs.
+|(not set)
+|Any
+|<<auto-generated-ids>>
+
+|imagesdir
+|Where image files are stored.
+|(not set) in my tests, ./images in text
+|Path
+|<<images>>
+
+|important-caption
+|Sets the text used to label IMPORTANT admonitions, if icons are not used.
+|Important
+|Any
+|<<admonition>>
+
+|indent
+|Normalize block indentation in code listings.
+|(not set)
+|Any (value not used)
+|<<normalize-block-indentation>>
+
+|lang
+|Set the `lang` attribute in the Docbook backend.
+|en
+|Any valid XML country code
+|
+
+|last-update-label
+|Text label for the "last updated" time in the footer.
+Unsetting it will remove the label and time from the footer.
+|Last updated
+|Any
+|<<footer-docinfo-files>>
+
+|linkcss
+|link to the stylesheet instead of embedding it.
+|(not set)
+|Any (value not used)
+|<<styling-the-html-with-css>>
+
+|mantitle
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manvolnum
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manname
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manpurpose
+|Metadata for manpage output.
+|(not set)
+|Any
+|<<man-pages>>
+
+|manname-title
+|Label for the program name liner in manpage outputs?
+|NAME
+|Any
+|?
+
+|max-include-depth
+|Not normally used. Should it be exposed?
+|64
+|-
+|-
```

Comment:
max-include-depth can only be set by the API, not the document. (this is a security feature)